### PR TITLE
The lorenining of Guns- Rifle Edition

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Companies-
Raummetall- MGs, Certain Rifles (Focus on MGs and Wacky Experimental rifles, T-12 is just there because RMML AR-12 sounds cool)
Kauser- ALF rifles- Skirmish Rifle, LMG, carbine (Focus on that rifle platform overall.)
SCA- Halo guns. (Yes.)
Bauer- Autoshotgun and Chambered sniper (Placeholder) 
KrugTech- Literally a company made to hold a patent for smartgun computing and optics royalty. That's it.

Will do the rest of rifles, SMGs and other MGs in the next edition, I just wanna keep this small for now.

## Why It's Good For The Game
Gun lore good, and screw it, we have a new nomelcuture, might aswell spice it up.

## Changelog
:cl:
add: Added new gun lore. Check the codex for cool reading.
add: New gun names, more nomenclature.
/:cl:

### CODE REVIEW

File: code/modules/codex/entries/guns_codex.dm [hunk 1]
```diff
@@ -252,34 +252,48 @@
...
 	associated_paths = list(/obj/item/weapon/gun/rifle/standard_assaultrifle)
-	lore_text = "The RRL-51 used to be the Marine Corps standard issue Rifle. It was known for it's rather heavy and large \
-	size but good accuracy at range. But it was eventually replaced by the AR-18 because of it's relatively poor performance at close range situations.<br><br>\
-	The RRL-51 (Corps designation AR-12) is still widely used by the Corps because of the sheer amount in surplus and because it's still a rather reliable weapon in the field.<br><br>\
-	The original RRL-51 was designed once news of a trial for the adoption of a caseless assault rifle to replace the current aging \
-	rifle were made public to all gun manufacturers, and eventually the RRL-51s design won out its competitors due to its ability to stay reliable in harsh conditions."
+	lore_text = "The Keckler and Hoch T-12 was created out of an order for a robust rifle using a caseless round, after the ergonomic and costly disaster of the 'Rifle of the future' Project. \
+	Ironically, the T-12 submitted by the very group that made the national disaster that the AR-11 (Oringally termed T-11.) ended up winning out all the tests. \
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:256]:
```suggestion
	Ironically, the T-12 submitted by the very group that made the national disaster that the AR-11 (originally designated the T-11) ended up winning out all the tests. \
```

File: code/modules/codex/entries/guns_codex.dm [hunk 2]
```diff
@@ -252,34 +252,48 @@
...
 
 /datum/codex_entry/standard_carbine
 	associated_paths = list(/obj/item/weapon/gun/rifle/standard_carbine)
-	lore_text = "The reason for the adoption of the ALF-4 carbine surfaced after repeated complaints of the unwieldiness of the AR-12 \
-	Rifle surfaced inside tight spaces, this then prompted the higher command of the TGMC to decide to adopt a new rifle within new barrel \
-	length designations due to the rising prevalence of fighting in close quarters situations inside spaceships and some colonies.<br><br>\
-	Eventually a selection process was made and the ALF-4 eventually won the trials and field testing soon ensured. Which eventually lead \
-	to great feedback from the troops supplied with it. This soon lead to it replacing the AR-12 Rifle as the standard issue firearm. \
-	With the designation AR-18. Though the AR-12 itself is still used widely due to availability."
+	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a simmilar manual of arms to appeal to simplify famaliarity and attract large scale buyers.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:263]:
```suggestion
	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a similar manual of arms to appeal to simplify familiarity and attract large scale buyers.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 3]
```diff
@@ -252,34 +252,48 @@
...
-	The RRL-51 (Corps designation AR-12) is still widely used by the Corps because of the sheer amount in surplus and because it's still a rather reliable weapon in the field.<br><br>\
-	The original RRL-51 was designed once news of a trial for the adoption of a caseless assault rifle to replace the current aging \
-	rifle were made public to all gun manufacturers, and eventually the RRL-51s design won out its competitors due to its ability to stay reliable in harsh conditions."
+	lore_text = "The Keckler and Hoch T-12 was created out of an order for a robust rifle using a caseless round, after the ergonomic and costly disaster of the 'Rifle of the future' Project. \
+	Ironically, the T-12 submitted by the very group that made the national disaster that the AR-11 (Oringally termed T-11.) ended up winning out all the tests. \
+	It was designed to be simple to use, dissasamble and basically survive every condition, however due to reports of barrel melting from the high use of polymers. It ended up being heavier than its ilk to compensate.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:257]:
```suggestion
	It was designed to be simple to use, disassemble and basically survive every condition, however due to reports of barrel melting from the high use of polymers. It ended up being heavier than its ilk to compensate.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 4]
```diff
@@ -252,34 +252,48 @@
...
-	to great feedback from the troops supplied with it. This soon lead to it replacing the AR-12 Rifle as the standard issue firearm. \
-	With the designation AR-18. Though the AR-12 itself is still used widely due to availability."
+	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a simmilar manual of arms to appeal to simplify famaliarity and attract large scale buyers.\
+	The Carbine variant, known as the 'ALF-51' focused on creating a small, compact package that could fire a rifle-sized round while still retaining good accuracy and shot placement.\
+	Orginally, it was meant to fire in burst-fire only, future production variants turned to feature full-auto after it showed that in testing, it would improve troop morale and confidence in the versatility in the rifle.\
+	The entire ALF-series ended up being panick bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:266]:
```suggestion
	The entire ALF-series ended up being panic bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 5]
```diff
@@ -252,34 +252,48 @@
...
-	Rifle (non-marine designation RRL-51) also began to suffer under prolonged combat due to overheating and low ammo count.<br><br>\
-	The ALF-8 itself is based off the ALF-4’s design. The only major differences are a heavier and longer barrel created to have a \
-	large amount of rounds go threw it in short periods of time and different mag wells to accept bigger magazines than it’s little brother. \
-	It was eventually adopted and designated the MG-42.<br><br>\
-	However these days it's been mostly replaced by the M56 smartgun system due to the M56s ability to use IFF and also self-load due to it’s \
-	autoloading systems. Removing the need of an assistant to carry the bulky ammo.<br><br>\
-	However, the MG-42 is still used since it's much easier to use than the special training required for the smartgun system. These days it's \
-	usually used when IFF isn't needed such as high visibility environments and defensive positions."
+	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a simmilar manual of arms to appeal to simplify famaliarity and attract large scale buyers.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:272]:
```suggestion
	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a similar manual of arms to appeal to simplify familiarity and attract large scale buyers.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 6]
```diff
@@ -252,34 +252,48 @@
...
+	The Machinegun variant, known as the 'ALF-22' focused on creating a versatile package consisting of a light squad support weapon, with a focus on mobility and accuarate fire over pure magazine size.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:273]:
```suggestion
	The Machinegun variant, known as the 'ALF-22' focused on creating a versatile package consisting of a light squad support weapon, with a focus on mobility and accurate fire over pure magazine size.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 7]
```diff
@@ -252,34 +252,48 @@
...
+	It ended up with a drum magazine, elongated heavy barrel for sustained firing, its overall shape being of a common rifle generally assists it in being easy to pickup and use, and can mount more underbarreled attachments compared to its peers in its class.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:274]:
```suggestion
	It ended up with a drum magazine, elongated heavy barrel for sustained firing, its overall shape being of a common rifle generally assists it in being easy to pickup and use, and can mount more underbarrel attachments compared to its peers in its class.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 8]
```diff
@@ -252,34 +252,48 @@
...
+	The entire ALF-series ended up being panick bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:275]:
```suggestion
	The entire ALF-series ended up being panic bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 9]
```diff
@@ -252,34 +252,48 @@
...
+	Eventually, they ended up becoming standard issue and a formal contract was signed, the Machinegun was christened the 'MG-42'.\
+	Most would say the large magazine capacity combined with ease of use and ability to lay down fire with good frontline potiential is the main advantage of this weapon. You will still be one the slower people in your group, however."
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:277]:
```suggestion
	Most would say the large magazine capacity combined with ease of use and ability to lay down fire with good frontline potential is the main advantage of this weapon. You will still be one the slower people in your group, however."
```

File: code/modules/codex/entries/guns_codex.dm [hunk 10]
```diff
@@ -252,34 +252,48 @@
...
+
+/datum/codex_entry/standard_skirmishrifle
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_skirmishrifle)
+	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a simmilar manual of arms to appeal to simplify famaliarity and attract large scale buyers.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:281]:
```suggestion
	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a similar manual of arms to appeal to simplify familiarity and attract large scale buyers.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 11]
```diff
@@ -252,34 +252,48 @@
...
+	The 'Skirmish' Rifle variant, known as the 'ALF-6' focused on creating a fullpower rifle, light and with a good blend of mobility, firerate and firepower.\
+	The rifle ended up being the rifle to truly kick off the ALF series, becoming famous for being a great blend of firepower, mobility and ease of use, however the system was known for tearing rounds like it was nothing.\
+	The entire ALF-series ended up being panick bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:284]:
```suggestion
	The entire ALF-series ended up being panic bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 12]
```diff
@@ -252,34 +252,48 @@
...
+	Eventually, they ended up becoming standard issue and a formal contract was signed, the Rifle was christened the 'AR-21'.\
+	Most would say it earned its term of a 'Skirmish' rifle due to use in constant Skrimishes, due to a low magazine size, but good overall mobility combined with firepower. Watch your magazine count and put the hurt downrange."
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:286]:
```suggestion
	Most would say it earned its term of a 'Skirmish' rifle due to use in constant skirmishes by frontline troops, due to a low magazine size, but good overall mobility combined with firepower. Watch your magazine count and put the hurt downrange."
```

File: code/modules/codex/entries/guns_codex.dm [hunk 13]
```diff
@@ -252,34 +252,48 @@
...
+
+/datum/codex_entry/standard_assaultrifle
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_assaultrifle)
+	lore_text = "The Keckler and Hoch 'T-11' was created out of a infamous bid called the 'Advanced Rifle of the Future' project, created out of a need to replace aging weapon stocks in the TerraGov Arsenal. \
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:290]:
```suggestion
	lore_text = "The Keckler and Hoch 'T-11' was created out of a infamous bid called the 'Rifle of the future' project, created out of a need to replace aging weapon stocks in the TerraGov arsenal. \
```

File: code/modules/codex/entries/guns_codex.dm [hunk 14]
```diff
@@ -252,34 +252,48 @@
...
+	The T-11 was created to be the ultimate weapon, being able to lay down fire like an MG with a high capacity, amazing burst fire capablity, specialized scope for long range damage, it was tested and destroyed the competition.\
+	It did amazing in field tests, and was effectively the best rifle overall. Most concerns were ergonomical, however it was pushed aside for a need of a new rifle, and money was already spent in making the entire thing, so getting a new competition would be too bothersome.\
+	Eventually however, these ergonomic failures lead to constant troop complaints due to the unergonomic nature of the rifle, troops who swore by it loved it, troops who hated it called it a useless brick.\
+	Eventually pushback got so large that a new bid was quickly placed within less than a decade, and the T-11, by now christened the 'AR-11'. Was considered a failure and mothballed.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:294]:
```suggestion
	Eventually pushback got so large that a new bid was quickly placed within less than a decade, and the T-11, by now christened the 'AR-11', was considered a failure and mothballed.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 15]
```diff
@@ -252,34 +252,48 @@
...
+	However, some have been pulled out of longterm storage, and recent spottings have shown modernizations such as removal of the side magazines, an overall more ergomic shape with less brick-like features, and a larger magazine.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:295]:
```suggestion
	However, some have been pulled out of long term storage, and recent spottings have shown modernizations such as removal of the side magazines, an overall more ergonomic shape with less brick-like features, and a larger magazine.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 16]
```diff
@@ -252,34 +252,48 @@
...
+	Most would say that the unusually large magazine capacity, amazing burst fire capablity allows for an interesting use of a support weapon. But its bulky and unusual shape leading a low amount of attachments can hamper it in combat."
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:296]:
```suggestion
	Most would say that the unusually large magazine capacity, amazing burst fire capability allows for an interesting use of a support weapon. But its bulky and unusual shape leading a low amount of attachments can hamper it in combat."
```
i swear you need sleep

File: code/modules/codex/entries/guns_codex.dm [hunk 17]
```diff
@@ -252,34 +252,74 @@
...
+	Most would say that the unusually large magazine capacity, amazing burst fire capability allows for an interesting use of a support weapon. But its bulky and unusual shape leading a low amount of attachments can hamper it in combat."
+
+/datum/codex_entry/standard_autosniper
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_autosniper)
+	lore_text = "The Kauser ALF series of firearms was born out of a desire to create a modular weapon design that could ergonomically use a similar manual of arms to appeal to simplify familiarity and attract large scale buyers.\
+	The Sniper variant, known as the 'ALF-1' focused on creating a long range marksman rifle, focused on combining both mobility and rate of fire with good overall firepower.\
+	This particular version of the rifle is modernized with a KTLD gun comptuer and vision scope to allow for quick IFF in hectic engagements, and corrections for the user to stop FF, unlike older variants, it requires no specialized training to use.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:302]:
```suggestion
	This particular version of the rifle is modernized with a KTLD gun computer and vision scope to allow for quick IFF in hectic engagements, and corrections for the user to stop FF, unlike older variants, it requires no specialized training to use.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 18]
```diff
@@ -252,34 +252,74 @@
...
+	The ALF-1 is basically a rifle with a heavier and longer barrel and a irremoveable scope, which sacrifices many of the traditional sniper elements for the normal familiarity of a normal everyday rifle, it is far less common than its peers in its family.\
+	The entire ALF-series ended up being panic bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
+	Eventually, they ended up becoming standard issue and a formal contract was signed, the Sniper was christened the 'GL-81'.\
+	Most would say that this rifle blends a feature of a normal rifle and sniper rifle, and has the ups and downs of both. Watch out in close quarters, and use the IFF and NVG features of the scope to your advantage.."
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:306]:
```suggestion
	Most would say that this rifle blends a feature of a normal rifle and sniper rifle, and has the ups and downs of both. Watch out in close quarters, and use the IFF and NVG features of the scope to your advantage..."
```

File: code/modules/codex/entries/guns_codex.dm [hunk 19]
```diff
@@ -252,34 +252,74 @@
...
+
+/datum/codex_entry/standard_smartmachinegun
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_smartmachinegun)
+	lore_text = "The Raummetall 'T-29' was made out of the heavy and unruly corpse of the 'T-28', an attempt to modernize and integrate a KTLD into the T-27 model machinegun. It ended in failure due to the overcomplexity of the system, and cost eventually made the project die..\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:310]:
```suggestion
	lore_text = "The Raummetall 'T-29' was made out of the heavy and unruly corpse of the 'T-28', an attempt to modernize and integrate a KTLD into the T-27 model machinegun. It ended in failure due to the overcomplexity of the system, and cost eventually made the project die.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 20]
```diff
@@ -252,34 +252,74 @@
...
+	The 'T-29' is effectively a continuation in spirit of the previous project, focusing on simplifying the IFF Computer systems. It only bears a outer resemblance to the T-27, outside of having the same basic systems.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:311]:
```suggestion
	The 'T-29' is effectively a continuation in spirit of the previous project, focusing on simplifying the IFF computer systems. It only bears a outer resemblance to the T-27, outside of having the same basic systems.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 21]
```diff
@@ -252,34 +252,74 @@
...
+	It has specialized systems to self-stabilize, and is made to be shoulder mounted and fired, as well as fired from lower positions, and uses its integral KTLD computer to self correct user errors, it however requires specialized training to use.\
+	The 'T-29 is effectively a incredibly accurate machinegun, it ended up being offered to the Marine Corps after the previous attempt to just modernize the older machineguns failed. It ended up being christened the 'MG-28' and entered service.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:313]:
```suggestion
	The 'T-29' is effectively a incredibly accurate machinegun, it ended up being offered to the Marine Corps after the previous attempt to just modernize the older machineguns failed. It ended up being christened the 'MG-28' and entered service.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 22]
```diff
@@ -252,34 +252,74 @@
...
+	Most would say that this machinegun makes you a walking turret, you'll be slow and unwieldy with no match when it comes to fire suppression on the mov."
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:314]:
```suggestion
	Most would say that this machinegun makes you a walking turret, you'll be slow and unwieldy with no match when it comes to fire suppression on the move."
```

File: code/modules/codex/entries/guns_codex.dm [hunk 23]
```diff
@@ -252,34 +252,74 @@
...
+
+/datum/codex_entry/standard_smartmmg
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_gpmg)
+	lore_text = "The Raummetall 'T-60' was designed out of a bid for a cheap and available machinegun that could move to several roles, be it fire support, vehicle mounted or sentry duties.\
+	It is basically a General Purpose Machinegun, meant to be used in about every scenario, usually not as well as a specialized gun, however it ended up being well known for its lightweightedness and mobility within its class.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:319]:
```suggestion
	It is basically a General Purpose Machinegun (GPMG), meant to be used in about every scenario, usually not as well as a specialized gun, however it ended up being well known for its lightness and mobility within its class.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 24]
```diff
@@ -252,34 +252,74 @@
...
+	Eventually, it won the bid due to its light weight, and low cost. Despite argugably being worse than most of the competition everywhere else outside of firerate. It was christened the 'MG-28' and entered service.\
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:320]:
```suggestion
	Eventually, it won the bid due to its light weight, and low cost for mass production. Despite arguably being worse than most of the competition everywhere else outside of firerate. It was christened the 'MG-28' and entered service.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 25]
```diff
@@ -252,34 +252,74 @@
...
+	Most would say that this machinegun makes you a walking dispenser of lead, you'll be slow and unwieldy with no match when it come to emptying a rain of lead on foe... Or friend."
```
Comment (XSlayer300) [inline code/modules/codex/entries/guns_codex.dm:321]:
```suggestion
	Most would say that this machinegun makes you a walking dispenser of lead, you'll be slow and unwieldy with no match when it come to emptying a rain of lead on foe... or friend."
```

**Comment by Pariah919** [comment]
Will do Rifles, then SMGs and other Smartgun guns, then specialist firearms. But I'd rather do this in batches.

File: code/modules/codex/entries/guns_codex.dm [hunk 26]
```diff
@@ -252,34 +252,74 @@
...
+	The 'T-29' is effectively a continuation in spirit of the previous project, focusing on simplifying the IFF computer systems. It only bears a outer resemblance to the T-27, outside of having the same basic systems.\
+	It has specialized systems to self-stabilize, and is made to be shoulder mounted and fired, as well as fired from lower positions, and uses its integral KTLD computer to self correct user errors, it however requires specialized training to use.\
+	The 'T-29' is effectively a incredibly accurate machinegun, it ended up being offered to the Marine Corps after the previous attempt to just modernize the older machineguns failed. It ended up being christened the 'MG-28' and entered service.\
+	Most would say that this machinegun makes you a walking turret, you'll be slow and unwieldy with no match when it comes to fire suppression on the move."
+
+/datum/codex_entry/standard_smartmmg
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_gpmg)
+	lore_text = "The Raummetall 'T-60' was designed out of a bid for a cheap and available machinegun that could move to several roles, be it fire support, vehicle mounted or sentry duties.\
```
Comment (TiviPlus) [inline code/modules/codex/entries/guns_codex.dm:318]: t60

File: code/modules/codex/entries/guns_codex.dm [hunk 27]
```diff
@@ -252,34 +252,74 @@
...
+	The entire ALF-series ended up being panic bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
+	Eventually, they ended up becoming standard issue and a formal contract was signed, the Sniper was christened the 'GL-81'.\
+	Most would say that this rifle blends a feature of a normal rifle and sniper rifle, and has the ups and downs of both. Watch out in close quarters, and use the IFF and NVG features of the scope to your advantage..."
+
+/datum/codex_entry/standard_smartmachinegun
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_smartmachinegun)
+	lore_text = "The Raummetall 'T-29' was made out of the heavy and unruly corpse of the 'T-28', an attempt to modernize and integrate a KTLD into the T-27 model machinegun. It ended in failure due to the overcomplexity of the system, and cost eventually made the project die.\
```
Comment (TiviPlus) [inline code/modules/codex/entries/guns_codex.dm:310]: T EVERYWHEER

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/codex/entries/guns_codex.dm [hunk 28]
```diff
@@ -252,34 +252,74 @@
 
 /datum/codex_entry/standard_assaultrifle
 	associated_paths = list(/obj/item/weapon/gun/rifle/standard_assaultrifle)
-	lore_text = "The RRL-51 used to be the Marine Corps standard issue Rifle. It was known for it's rather heavy and large \
-	size but good accuracy at range. But it was eventually replaced by the AR-18 because of it's relatively poor performance at close range situations.<br><br>\
-	The RRL-51 (Corps designation AR-12) is still widely used by the Corps because of the sheer amount in surplus and because it's still a rather reliable weapon in the field.<br><br>\
-	The original RRL-51 was designed once news of a trial for the adoption of a caseless assault rifle to replace the current aging \
-	rifle were made public to all gun manufacturers, and eventually the RRL-51s design won out its competitors due to its ability to stay reliable in harsh conditions."
+	lore_text = "The Keckler and Hoch T-12 was created out of an order for a robust rifle using a caseless round, after the ergonomic and costly disaster of the 'Rifle of the future' Project. \
```
Comment (Pariah919) [inline code/modules/codex/entries/guns_codex.dm:255]:
```suggestion
	lore_text = "The Keckler and Hoch M-12 was created out of an order for a robust rifle using a caseless round, after the ergonomic and costly disaster of the 'Rifle of the future' Project. \
```

File: code/modules/codex/entries/guns_codex.dm [hunk 29]
```diff
@@ -252,34 +252,74 @@
...
+	Ironically, the T-12 submitted by the very group that made the national disaster that the AR-11 (originally designated the T-11) ended up winning out all the tests. \
```
Comment (Pariah919) [inline code/modules/codex/entries/guns_codex.dm:256]:
```suggestion
	Ironically, the M-12 submitted by the very group that made the national disaster that the AR-11 (originally designated the M-11) ended up winning out all the tests. \
```

File: code/modules/codex/entries/guns_codex.dm (hunk 27 above)
Comment (Pariah919) [inline code/modules/codex/entries/guns_codex.dm:310]:
```suggestion
	lore_text = "The Raummetall 'M-29' was made out of the heavy and unruly corpse of the 'T-28', an attempt to modernize and integrate a KTLD into the M-27 model machinegun. It ended in failure due to the overcomplexity of the system, and cost eventually made the project die.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 30]
```diff
@@ -252,34 +252,74 @@
...
+	The entire ALF-series ended up being panic bought in mass by specialized troops in the Marine Corps, generally Peacekeeper sections or Air Assault sections, it eventually spread by osmosis in popularity.\
+	Eventually, they ended up becoming standard issue and a formal contract was signed, the Sniper was christened the 'GL-81'.\
+	Most would say that this rifle blends a feature of a normal rifle and sniper rifle, and has the ups and downs of both. Watch out in close quarters, and use the IFF and NVG features of the scope to your advantage..."
+
+/datum/codex_entry/standard_smartmachinegun
+	associated_paths = list(/obj/item/weapon/gun/rifle/standard_smartmachinegun)
+	lore_text = "The Raummetall 'T-29' was made out of the heavy and unruly corpse of the 'T-28', an attempt to modernize and integrate a KTLD into the T-27 model machinegun. It ended in failure due to the overcomplexity of the system, and cost eventually made the project die.\
+	The 'T-29' is effectively a continuation in spirit of the previous project, focusing on simplifying the IFF computer systems. It only bears a outer resemblance to the T-27, outside of having the same basic systems.\
```
Comment (Pariah919) [inline code/modules/codex/entries/guns_codex.dm:311]:
```suggestion
	The 'M-29' is effectively a continuation in spirit of the previous project, focusing on simplifying the IFF computer systems. It only bears a outer resemblance to the M-27, outside of having the same basic systems.\
```

File: code/modules/codex/entries/guns_codex.dm [hunk 31]
```diff
@@ -252,34 +252,74 @@
...
+	It has specialized systems to self-stabilize, and is made to be shoulder mounted and fired, as well as fired from lower positions, and uses its integral KTLD computer to self correct user errors, it however requires specialized training to use.\
+	The 'T-29' is effectively a incredibly accurate machinegun, it ended up being offered to the Marine Corps after the previous attempt to just modernize the older machineguns failed. It ended up being christened the 'MG-28' and entered service.\
```
Comment (Pariah919) [inline code/modules/codex/entries/guns_codex.dm:313]:
```suggestion
	The 'M-29' is effectively a incredibly accurate machinegun, it ended up being offered to the Marine Corps after the previous attempt to just modernize the older machineguns failed. It ended up being christened the 'MG-28' and entered service.\
```

File: code/modules/codex/entries/guns_codex.dm (hunk 26 above)
Comment (Pariah919) [inline code/modules/codex/entries/guns_codex.dm:318]:
```suggestion
	lore_text = "The Raummetall 'M-60' was designed out of a bid for a cheap and available machinegun that could move to several roles, be it fire support, vehicle mounted or sentry duties.\
```

**Review by Pariah919 — COMMENTED** [review]
T-PTSD from maintainers

File: code/modules/projectiles/guns/rifles.dm [hunk 32]
```diff
@@ -1118,8 +1118,9 @@
 //SR-81 Auto-Sniper
 
 /obj/item/weapon/gun/rifle/standard_autosniper
-	name = "\improper SR-81 automatic sniper rifle"
-	desc = "The SR-81 is the TerraGov Marine Corps's automatic sniper rifle usually married to it's iconic NVG/KTLD scope combo. It's users use it for it's high rate of fire for it's class, and has decent performance in any range. Uses 8.6x70mm caseless with specialized pressures for IFF fire."
+	name = "\improper SR-81 Kauser-KT automatic sniper rifle"
+	desc = "The Kauser-KT SR-81 is the TerraGov Marine Corps's automatic sniper rifle usually married to it's iconic NVG/KTLD scope combo. It's users use it for it's high rate of fire for it's class, and has decent performance in any range. Uses 8.6x70mm caseless with specialized pressures for IFF fire."
+
```
Comment (Pariah919) [inline code/modules/projectiles/guns/rifles.dm:1123]:
```suggestion
```

**Review by BraveMole — APPROVED** [review]

**Pull request merged**